csvqb/csvqb/models/cube/cube.py

```diff
@@ -45,14 +43,14 @@ def __init__(self,
 
     @staticmethod
     def from_dict(config: dict) -> "CubeMetadata":
+        publisher = get_with_func_or_none(config, "publisher", URI)
         return CubeMetadata(
             get_from_dict_ensure_exists(config, "title"),
             uri_safe_identifier=get_from_dict_ensure_exists(config, "id"),
-            base_uri=config.get("baseUri"),
             summary=config.get("summary"),
             description=config.get("description"),
-            creator=get_with_func_or_none(config, "creator", URI),
-            publisher=get_with_func_or_none(config, "publisher", URI),
+            creator=publisher,
```

Comment:
I don't like how the creator is assigned the value of the publisher by default. Is this standard practise @robons?

Reply:
There's no field in the [info.json schema](https://github.com/GSS-Cogs/family-schemas/blob/main/dataset-schema.json) to differentiate between the publisher/creator.

Reply:
[How we used to do this in gss-utils](https://github.com/GSS-Cogs/gss-utils/blob/933bc0de73cae282f82bc43b2338e0c720adb948/gssutils/metadata/pmdcat.py#L65).